### PR TITLE
Homepage Revamp

### PR DESCRIPTION
**Pull request by jackyzha0**
# things
* made hacker countdown fetch from firebase
* made day/days logic actually work lmao
* fixed progress bar logic
* add twitch embed

Screen Shot 2020-09-01 at 5 29 57 PM

Screen Shot 2020-09-01 at 5 30 16 PM

### CODE REVIEW

File: src/components/Livestream.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import styled, { keyframes } from 'styled-components'
+import { Card } from '../components/Common.js'
+import { H2 } from '../components/Typography'
+
+const TwitchFrame = styled.iframe`
+  border-radius: 5px;
+  border: none;
+`
+
+const LivestreamContainer = styled(Card)`
```
Comment (ianmah) [inline src/components/Livestream.js:11]: Seems like we have a lot of names for a container, sometimes its `Container`, or `Div`, or `Card`

File: src/components/Livestream.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        src="https://player.twitch.tv/?channel=nwplus&parent=localhost&parent=live.nwplus.io"
+        height="720"
+        width="1280"
+        frameborder="0"
+        scrolling="no"
+        allowfullscreen="true">
+      </TwitchFrame>
+      <LeftAlign>
+        <H2><LiveDot /> nwPlus - Live</H2>
+                View on <a href="https://www.twitch.tv/nwplus" target="_blank" rel="noopener noreferrer">Twitch</a>
```
Comment (ianmah) [inline src/components/Livestream.js:55]: Whats the usecase for viewing on twitch?

File: src/containers/HackerCountdown.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+export default () => {
+  const [start, setStart] = useState(new Date())
+  const [end, setEnd] = useState(new Date())
+
+  function toDateTime(secs) {
+    var t = new Date(1970, 0, 1);
+    t.setSeconds(secs);
+    return t;
+  }
```
Comment (ianmah) [inline src/containers/HackerCountdown.js:14]: I was under the impression we weren't using `timestamp` in firebase? only ISO date strings?

File: src/components/Livestream.js (hunk 1 above)
Comment (jackyzha0) [reply src/components/Livestream.js:11]: what do you suggest we standardize it to

File: src/components/Livestream.js (hunk 2 above)
Comment (jackyzha0) [reply src/components/Livestream.js:55]: image

I just wanted something to put here LOL should I just omit it then

File: src/containers/HackerCountdown.js (hunk 3 above)
Comment (jackyzha0) [reply src/containers/HackerCountdown.js:14]: ah shoot forgot about that convo, I'll fix

File: src/components/Livestream.js (hunk 1 above)
Comment (ianmah) [reply src/components/Livestream.js:11]: I think if its actually using `CardLike` or the actual card component use Card, everything else use Container?

Comment (ianmah) [reply src/components/Livestream.js:11]: Mind renaming `TimeDiv` and anything else? :*

Comment (jackyzha0) [reply src/components/Livestream.js:11]: will do

File: src/components/Livestream.js (hunk 2 above)
Comment (ianmah) [reply src/components/Livestream.js:55]: uhmmm i guess omit for now

**Review by ianmah — APPROVED** [review]

**Pull request merged**